pkg/services/sqlstore/org_users.go

```diff
@@ -142,6 +142,68 @@ func GetOrgUsers(query *models.GetOrgUsersQuery) error {
 	return nil
 }
 
+func (ss *SQLStore) SearchOrgUsers(query *models.SearchOrgUsersQuery) error {
+	query.Result = models.SearchOrgUsersQueryResult{
+		OrgUsers: make([]*models.OrgUserDTO, 0),
+	}
+
+	sess := x.Table("org_user")
+	sess.Join("INNER", x.Dialect().Quote("user"), fmt.Sprintf("org_user.user_id=%s.id", x.Dialect().Quote("user")))
+
+	whereConditions := make([]string, 0)
+	whereParams := make([]interface{}, 0)
+
+	whereConditions = append(whereConditions, "org_user.org_id = ?")
+	whereParams = append(whereParams, query.OrgID)
+
+	if query.Query != "" {
+		queryWithWildcards := "%" + query.Query + "%"
+		whereConditions = append(whereConditions, "(email "+dialect.LikeStr()+" ? OR name "+dialect.LikeStr()+" ? OR login "+dialect.LikeStr()+" ?)")
+		whereParams = append(whereParams, queryWithWildcards, queryWithWildcards, queryWithWildcards)
+	}
+
+	if len(whereConditions) > 0 {
+		sess.Where(strings.Join(whereConditions, " AND "), whereParams...)
+	}
+
+	if query.Limit > 0 {
+		offset := query.Limit * (query.Page - 1)
+		sess.Limit(query.Limit, offset)
+	}
+
+	sess.Cols(
+		"org_user.org_id",
+		"org_user.user_id",
+		"user.email",
+		"user.name",
+		"user.login",
+		"org_user.role",
+		"user.last_seen_at",
+	)
+	sess.Asc("user.email", "user.login")
+
+	if err := sess.Find(&query.Result.OrgUsers); err != nil {
+		return err
+	}
+
+	// get total count
+	orgUser := models.OrgUser{}
+	countSess := x.Table("org_user")
+
+	if len(whereConditions) > 0 {
+		countSess.Where(strings.Join(whereConditions, " AND "), whereParams...)
+	}
+
+	count, err := countSess.Count(&orgUser)
+	query.Result.TotalCount = count
+
+	for _, user := range query.Result.OrgUsers {
+		user.LastSeenAtAge = util.GetAgeString(user.LastSeenAt)
+	}
+
+	return err
```

Comment:
There's no reason to not return straight away on error, is there?
```suggestion
	count, err := countSess.Count(&orgUser)
	if err != nil {
		return err
	}
	query.Result.TotalCount = count

	for _, user := range query.Result.OrgUsers {
		user.LastSeenAtAge = util.GetAgeString(user.LastSeenAt)
	}

	return nil
```